src/shogun/features/CombinedDotFeatures.cpp

```diff
@@ -347,5 +356,7 @@ void CCombinedDotFeatures::init()
 	SG_ADD(
 	    &num_vectors, "num_vectors", "Total number of vectors.");
 	SG_ADD(&feature_array, "feature_array", "Feature array.");
+	// FIXME
+	// SG_ADD(&feature_weights, "feature_weights", "Subfeature weights.");
```

Comment:
whats here?

Reply:
The weights vector should be added as parameter, but std::vector isn't supported yet.

Reply:
I think it is! (Not via SG_ADD though, you have to use `watch_param`

Reply:
yup, you're right